internal/dag/builder_test.go

```diff
@@ -10913,7 +10945,26 @@ func TestDAGInsert(t *testing.T) {
 			objs: []interface{}{
 				proxy108, proxy108a, proxy108b, s1, s12, s13,
 			},
-			want: listeners(),
```

Comment:
I'd call out this behavior change explicitly in the changelog (i.e. that the first include with a given set of conditions is programmed/not orphaned, only subsequent ones with dupe conditions are now not programmed).

Reply:
let me know what you think!